### DISCUSSION TITLE
Circuit Recipes

### DISCUSSION THREAD

**Discussion by HbmMods** [HbmMods/Hbm-s-Nuclear-Tech-GIT#448] [General]
Circuits are annoying as hell to make because of the required "upgrading" of previous tiers. Do yall think circuits should have from-scratch recipes that don't require the previous tier circuit instead?

**Comment by sdddddf80** [comment]
Honestly，this “upgrading”makes me feel that I am building a true production line to produce greater circuits. If any circuits just need the basic  
circuits to craft, than the circuits production line won't exist anymore, which may be more convenient，but also it reduces the happiness.
I think if we have to make circuits have from-scratch recipes that don't require the previous tier circuit instead, than at least make the higher tier circuits require more advanced machines to craft, like the factory, as a furnace with multi block structure is meaningless when you have electric arc furnace. It should be used for something complex, like assemble circuits.

**Comment by Nos74** [comment]
yeah circuits are pretty ass to make as they are. individualizing each tier and locking them behind already progression-locked materials sounds reasonable (that way once you reach a certain level of industry you can just shit out the circuits that you need)

**Comment by ncr-t** [comment]
what about keeping both ? maybe rebalance it so the new recipe needs more advanced materials ?

**Comment by Vaern** [comment]
Maybe you could take a more middle-ground approach. Maybe you'd need to make an actual circuitboard or some other base, and then assemble the respective circuit with the progression-locked materials. That way, there would still be reason for an assembly line esque approach (especially considering you'd still need the raw materials for the circuit) whilst also reducing the CBT and microcrafting overall.

**Comment by UFFR** [comment]
That would reduce the cost of later tier circuits significantly. Without 
a re-balancing of the recipes, there may some issues with certain usage 
recipes, particularly ones that have "optional" circuit tiers like the 
missile targeting circuits. Regardless, this idea should be continued as 
it is undoubtedly tedious and cumbersome to bulk produce late-tier circuits.

-- 
UFFR
